gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -2032,6 +2032,16 @@ paths:
         parameters:
             - $ref: "#/components/parameters/envIdParam"
             - $ref: "#/components/parameters/apiIdParam"
+            - in: query
+              name: from
+              schema:
+                type: integer
+              description: The start timestamp of sample
```

Comment:
As we are also using it for HealthCheck stuff what about creating a dedicated schema to use a ref like envId & apiId above?